.github/workflows/docs.yaml

```diff
@@ -59,6 +59,8 @@ jobs:
         run: |
           make -j deps-ci
           make -j build-docs
+        env:
+          DEPLOYER_PRIVATE_KEY: 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
please add a comment explaining this is a test key from foundry, so there is no confusion in the future